src/ciphersuite/mod.rs

```diff
@@ -51,6 +56,8 @@ pub enum HKDFError {
     InvalidLength,
 }
 
+// TODO: remove these and use the proper types from HPKE.
+
```

Comment:
I was wondering why we weren't doing that. The functions below seem to wrap around the everycrypt HPKE implementation without really doing anything.

Reply:
We will do that as soon as the other PRs are merged. See https://github.com/raphaelrobert/maelstrom/pull/39#issue-494802567
This needs another round of clean up but there's too much in flight and the change is too intrusive.